cdap-client-tests/src/test/java/co/cask/cdap/client/AdapterClientTestRun.java

```diff
@@ -86,6 +86,12 @@ public void testAdapters() throws Exception {
     adapterClient.waitForStatus(adapterName, AdapterStatus.STOPPED, 30, TimeUnit.SECONDS);
     adapterClient.start(adapterName);
     adapterClient.waitForStatus(adapterName, AdapterStatus.STARTED, 30, TimeUnit.SECONDS);
+
+    // The following test makes sure that if the request is malformed by the client, it will result in an error
```

Comment:
I don't think you need to specify this comment (describes a no longer existing bug). Just testing it is fine.